include/swift/AST/Decl.h

```diff
@@ -5635,9 +5638,13 @@ class ParamDecl : public VarDecl {
 
     /// Whether or not this parameter is '_const'.
     IsCompileTimeConst = 1 << 1,
+
+    // FIXME: it's not on the name but on the type
+    /// Whether or not this parameter is a '_local DistributedActor' reference.
+    IsDistributedKnownToBeLocal = 1 << 2,
```

Comment:
not actually sure if we need it, we can ride on the attr

Reply:
It should be here, because we might end up inferring it in some cases (e.g., for a closure).

Reply:
Aha, I see, thank you!